libs/blocks/section-metadata/section-metadata.js

```diff
@@ -12,11 +12,38 @@ function handleBackground(div, section) {
   }
 }
 
-function handleStyle(div, section) {
-  const value = div.textContent.toLowerCase();
-  const styles = value.split(', ').map((style) => style.replaceAll(' ', '-'));
+function handleStyle(div, section, customs = []) {
+  const value = div.textContent?.toLowerCase();
+  let styles = value.split(', ').map((style) => style.replaceAll(' ', '-')).filter(i => i !== '');
   if (section) {
-    section.classList.add(...styles);
+    if (customs.length) styles = [...styles, ...customs];
+    section.classList.add(...styles.filter(i => i !== ''));
+  }
+}
+
+function handleColumns(value, values) {
+  if (!value.includes('up')) return { columns: values, offset: null, upClass: null };
+  const ups = { 'two': 6, 'three': 4, 'four': 3, 'five': 2, 2: 6, 3: 4, 4: 3, 5: 2 };
+  const up = values.find(i => i.includes('up'));
```

Comment:
Having `value` and `values` is kind of confusing. Could you just include `values`?
```suggestion
function handleColumns(values) {
  const up = values.find(i => i.includes('up'));
  if (!up) return { columns: values, offset: null, upClass: null };
  const ups = { 'two': 6, 'three': 4, 'four': 3, 'five': 2, 2: 6, 3: 4, 4: 3, 5: 2 };
```

Reply:
Thanks 👍🏻